src/modules/clusters/cluster.service.ts

```diff
@@ -31,6 +31,63 @@ export class ClusterService {
     return this._clusterRepository.find(options);
   }
 
+  async countActive(): Promise<number> {
+    return this._clusterRepository.count({
+      where: {
+        isLiquidated: false,
+        burnRate: Not(0),
+      },
+    });
+  }
+
+  async countLiquidatable(): Promise<any> {
+    const minimumBlocksBeforeLiquidation =
+      await this._retryService.getWithRetry(
+        Web3Provider.minimumBlocksBeforeLiquidation,
+      );
+    const currentBlockNumber = +(await this._retryService.getWithRetry(
+      Web3Provider.currentBlockNumber,
+    ));
+
+    const toLiquidate = await this.findBy({
+      where: {
+        isLiquidated: false,
+        burnRate: Not(0),
+        liquidationBlockNumber: LessThanOrEqual(
+          currentBlockNumber + +minimumBlocksBeforeLiquidation,
+        ),
+      },
+    });
+
+    return toLiquidate.reduce(
```

Comment:
Should check the logic here why some of clusters are liquidatable but not liquidated. See comment:
https://discord.com/channels/@me/945613860293976065/1127601410029125702